website/source/docs/providers/template/index.html.markdown

```diff
@@ -25,7 +25,7 @@ Use the navigation to the left to read about the available resources.
 ```
 # Template for initial configuration bash script
 resource "template_file" "init" {
-	filename = "init.tpl"
+	template = "init.tpl"
```

Comment:
This change is not quite correct: the deprecated version uses `filename = "init.tpl"`, but the new version takes the *content* of the template in the variable which also allows inline templates in the form of heredocs (for example). Consequently in this example you'd likely use a the file interpolation function like this: `template = "${file("init.tpl")}".